gems/pending/metadata/linux/LinuxUtils.rb

```diff
@@ -141,6 +141,44 @@ def self.parse_systemctl_list(lines)
       end.compact!
     end
 
+    def self.collect_interface(interfaces, interface)
```

Comment:
The variable names and perhaps the method name make this very unclear to me here.  What are you collecting (when I hear collect, I think `.map`, but that's not what's happening here I don't think)?  and how does the single item relate to the collection?